features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -3285,4 +3285,11 @@
         {% endfor %}
     </OrganizationLevelEmailBrandingFallbacks>
 
+    <!-- Claim Management Configuration. -->
+    {% if claim_mgt.bind_to_primary_userstore is defined %}
+    <ClaimManagement>
```

Comment:
I am not clear why we added this configuration. Are we planning to enable this only in the Asgardeo?